frontend/src/pages/users/UserDetail.tsx

```diff
@@ -48,6 +48,12 @@ export default function VendorAdd() {
           setIsModifiable(!checkForSelfEditandAdminEdit(response.username));
           setUserName(response.username);
           setIsAdmin(response.is_staff);
+          setIsPageDeleteable(
+            !(
+              response.username === "admin" ||
+              id === localStorage.getItem("userID")
```

Comment:
This looks good, but there are a couple other requirements that I just wanted to ask about to make sure I'm up to date

1. "The admin account is special and cannot lose adminstrator privilege." I think this would just be disabling the toggle for the admin account?
2. The admin account username cannot be changed

Reply:
Yes the toggle is disabled. Right above this is a method I had made that makes the page disabled for editing.
Currently, no user that uses the website can change their name. I didn't see the requirement as to if they can. So admin's name can't be changed.

Reply:
Didn't realize user's couldn't change their name, that's my bad. And sounds good on the admin toggle, looks like everything is covered.